### PR TITLE
Add parallel runs for schemas validator 

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This PR needs to be used with [the relevant validator changes](https://github.com/ONSdigital/eq-questionnaire-validator/pull/252). It adds parallelism to our schemas validator runs.

### How to review
Use `make` commands, the branch of the validator is set temporarily so FastAPI version will be used.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
Will need a similar change in the questionnaire schemas repo 👍

**Review by liamtoozer — APPROVED** [review]
Looks good to me 👍  Just a couple of minor/cosmetic comments

File: scripts/validate_test_schemas.sh [hunk 1]
```diff
@@ -36,30 +39,37 @@ fi
...
 passed=0
+exit=0
 
 file_path_name=$(find "$file_path" -name '*.json')
 
-for schema in ${file_path_name}; do
-
-    result="$(curl -s -w 'HTTPSTATUS:%{http_code}' -X POST -H "Content-Type: application/json" -d @"$schema" http://localhost:5001/validate | tr -d '\n')"
+validate() {
+    result="$(curl -s -w 'HTTPSTATUS:%{http_code}' -X POST -H "Content-Type: application/json" -d @"$1" http://localhost:5001/validate | tr -d '\n')"
```
Comment (liamtoozer) [inline scripts/validate_test_schemas.sh:47]: Is it worth assigning `$1` to `schema` var at the top of this function?

File: scripts/validate_test_schemas.sh [hunk 2]
```diff
@@ -36,30 +39,37 @@ fi
 echo "--- Testing Schemas in $file_path ---"
 failed=0
 passed=0
+exit=0
```
Comment (liamtoozer) [inline scripts/validate_test_schemas.sh:42]: Can probably remove `exit=0` here as it's declared & initialised a little further up?

**Review by rmccar — APPROVED** [review]

**Pull request merged**